content/v2.0/reference/cli/influx/secret/delete.md

```diff
@@ -0,0 +1,28 @@
+---
+title: influx secret delete
+description: The 'influx secret delete' command deletes secrets.
+menu:
+  v2_0_ref:
+    name: influx secret delete
+    parent: influx secret
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret delete` command deletes secrets.
+
+## Usage
+```
+influx secret delete [flags]
+influx secret delete [command]
+```
+
+## Flags
+| Flag           | Description                 | Input type |
+|:----           |:-----------                 |:----------:|
+| `-h`, `--help` | Help for `delete`           |            |
```

Comment:
@sanderson, Help for the `secret delete` command...or Help for `secret delete`? -update across the board for other commands